tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that systemd timer is used for time synchronization.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data = get_test_suite_data();
+    select_console 'root-console';
+    record_info("Check yast-timesync.timer", "Check if file /etc/systemd/system/yast-timesync.timer exists and has expected configuration.");
+    assert_script_run('ls /etc/systemd/system/yast-timesync.timer');
+    my $conf_file = script_output("cat /etc/systemd/system/yast-timesync.timer");
+    $conf_file =~ /OnUnitActiveSec=(?<interval>(\S+))min/;
+    my $expected_interval = $test_data->{profile}->{'ntp-client'}->{ntp_sync};
+    assert_equals($expected_interval, $+{interval}, "The interval in yast-timesync configuration file is not the expected one.");
+    # Default value for first time sync is 1 minute after boot. Module pauses in order to be able to track the sync in the logs.
+    sleep 60;
```

Comment:
I thought about this but what complicates it a bit is that the commands take time to be typed and while typing, one time sync could run, so I have to measure exact typing time, then check if sync happens during typing or I should wait and then get time correction. Loop solution looks less error prone. Also, if someone sets a big interval the module will take long time either way. About sleep time, yes I can get it from conf file. Will do.

Reply:
Changed it to check logs according to expected sync time.

Reply:
Ok, sorry for not commenting on Friday. I have realized that one time sync cannot be predicted, due to significant inaccuracy. Following, you can see what the actual interval looks like in case of $expected_interval=1,2,3,4:
http://falafel.suse.cz/tests/505#step/verify_systemd_timesync/10
http://falafel.suse.cz/tests/506#step/verify_systemd_timesync/10
http://falafel.suse.cz/tests/507#step/verify_systemd_timesync/10
http://falafel.suse.cz/tests/508#step/verify_systemd_timesync/10

I get similar results on a VM on my laptop. I am not sure if I should file a bug for this. 
   As for the deviation fix in the test, I returned back to the loop idea as it's the safest one. Just added a wait of $expected_interval*30, again I cannot add a wait of the $expected_interval*60 because this is not accurate. Other solution I can think of would be not to include the particular check at all. WDYT?

Reply:
oh! good catch! could you check that with yast developers? If we need to guess too much or create difficult loop to read, IMO despite I like the idea to check this, I would put aside for the moment.

Reply:
I contacted the yast developer about this, waiting for answer. I also removed the deviation fix check from the test.